index.js

```diff
@@ -3,12 +3,15 @@
 module.exports = pixelmatch;
 
 const defaultOptions = {
-    threshold: 0.1,         // matching threshold (0 to 1); smaller is more sensitive
-    includeAA: false,       // whether to skip anti-aliasing detection
-    alpha: 0.1,             // opacity of original image in diff ouput
-    aaColor: [255, 255, 0], // color of anti-aliased pixels in diff output
-    diffColor: [255, 0, 0],  // color of different pixels in diff output
-    diffMask: false         // draw the diff over a transparent background (a mask)
+    threshold: 0.1,             // matching threshold (0 to 1); smaller is more sensitive
+    includeAA: false,           // whether to skip anti-aliasing detection
+    differentiateDiffs: false,  // whether to differentiate diff color between img1 and img2
```

Comment:
This feels a little goofy to do in addition to adding new `diffColor1` and `diffColor2` arguments.

Reply:
Yes, I think this could be simplified — how about only adding `diffColorAlt`, which is `null` by default, and implicitly turns on differentiating when given, being used as the second color (while the first is `diffColor`)? That way we'll have just 2 diff color options instead of 4.